fineract-provider/src/main/java/org/apache/fineract/portfolio/delinquency/api/DelinquencyApiResourceSwagger.java

```diff
@@ -0,0 +1,116 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.portfolio.delinquency.api;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public final class DelinquencyApiResourceSwagger {
+
+    private DelinquencyApiResourceSwagger() {}
+
+    @Schema(description = "GetDelinquencyRangesResponse")
+    public static final class GetDelinquencyRangesResponse {
+
+        private GetDelinquencyRangesResponse() {}
+
+        @Schema(example = "1")
+        public Integer id;
+        @Schema(example = "Delinquent 1")
+        public String classification;
+        @Schema(example = "1")
+        public Long minimumAgeDays;
+        @Schema(example = "3")
+        public Long maximumAgeDays;
+    }
+
+    @Schema(description = "GetDelinquencyBucketsResponse")
+    public static final class GetDelinquencyBucketsResponse {
+
+        private GetDelinquencyBucketsResponse() {}
+
+        @Schema(example = "1")
+        public Integer id;
+        @Schema(example = "Delinquent Bucket Set 1")
+        public String name;
+    }
+
+    @Schema(description = "PostDelinquencyRangeRequest")
+    public static final class PostDelinquencyRangeRequest {
+
+        private PostDelinquencyRangeRequest() {}
+
+        @Schema(example = "Delinquent 1")
+        public String classification;
+        @Schema(example = "1")
+        public Integer minimumAgeDays;
+        @Schema(example = "3")
+        public Integer maximumAgeDays;
+        @Schema(example = "en")
+        public String locale;
+    }
+
+    @Schema(description = "PostDelinquencyRangeResponse")
+    public static final class PostDelinquencyRangeResponse {
+
+        private PostDelinquencyRangeResponse() {}
+
+        @Schema(example = "1")
+        public Integer resourceId;
+    }
+
+    @Schema(description = "PutDelinquencyRangeResponse")
+    public static final class PutDelinquencyRangeResponse {
+
+        private PutDelinquencyRangeResponse() {}
+
+        @Schema(example = "1")
+        public Integer resourceId;
+        public PostDelinquencyRangeRequest changes;
+    }
+
+    @Schema(description = "PostDelinquencyBucketRequest")
+    public static final class PostDelinquencyBucketRequest {
+
+        private PostDelinquencyBucketRequest() {}
+
+        @Schema(example = "Delinquent 1")
+        public String name;
+        @Schema(example = "[1,2,3]")
+        public Long[] ranges;
+    }
+
+    @Schema(description = "PostDelinquencyBucketResponse")
+    public static final class PostDelinquencyBucketResponse {
+
+        private PostDelinquencyBucketResponse() {}
+
+        @Schema(example = "1")
+        public Integer resourceId;
+    }
+
+    @Schema(description = "PutDelinquencyBucketResponse")
+    public static final class PutDelinquencyBucketResponse {
+
+        private PutDelinquencyBucketResponse() {}
+
+        @Schema(example = "1")
+        public Integer resourceId;
+    }
+
```

Comment:
@josehernandezfintecheandomx Delete bucket request&response or delete range request&response is missing